flepimop/gempyor_pkg/src/gempyor/inference.py

```diff
@@ -563,6 +623,21 @@ def one_simulation(
         sim_id2load: int = None,
         parallel=False,
     ):
```

Comment:
```suggestion
    ) -> Literal[0]:
```

Based on the `Returns:` below.